x-pack/legacy/plugins/upgrade_assistant/public/np_ready/application/components/tabs/checkup/deprecations/reindex/polling_service.test.ts

```diff
@@ -6,9 +6,14 @@
 
 import { mockClient } from './polling_service.test.mocks';
 
-import { ReindexStatus, ReindexStep } from '../../../../../../common/types';
+import { ReindexStatus, ReindexStep } from '../../../../../../../../common/types';
 import { ReindexPollingService } from './polling_service';
 
+const getCoreHttpMock = (path?: string) =>
```

Comment:
You can also use core's mock from `src/core/public/http/http_service.mock.ts` (unlike code, importing mocks from core is whitelisted and encouraged). When you use just a single function it's not such a big deal, but since core's mock is API complete, switching might save you time if you start using other functions.

Reply:
I did see this, wasn't sure whether we could/should import, thanks for clarifying!